app/src/androidTest/java/com/epfl/drawyourpath/database/MockDatabaseTest.kt

```diff
@@ -55,6 +56,26 @@ class MockDatabaseTest {
         assertEquals(test, false)
     }
 
+    /**
+     * Test if a tournamentId present in the database is returned as present
+     */
+    @Test
```

Comment:
We might add one test for the failing database as well for coverage. Maybe also for `delete`, `add`, ...  operations on tournament.

Reply:
If you think about the `MockNonWorkingDatabase()` then the tests are in the related test class `MockNonWorkingDatabaseTest`, are you talking about other failing scenarios ?

Reply:
Yes that's what I meant. It's all good then.